src/js/common/validator.js

```diff
@@ -572,4 +596,24 @@ var fluid  = fluid  || {};
         });
         return requiredChildren;
     };
+
+    // A global component that validates material using the above static functions.  Handles the expensive initial
+    // compilation of the schema.  For performance reasons, this is not itself a schema-validated component.
+    fluid.defaults("gpii.schema.validator", {
+        gradeNames: ["fluid.component", "fluid.resolveRootSingle"],
+        singleRootType: "gpii.schema.validator",
+        members: {
+            validatorsByHash: {
```

Comment:
Provide an operation to evict validators on request otherwise the user has no option but for them to grow without limit!

Reply:
Agreed, and done and tested.